src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -943,6 +943,11 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                     body = BindMethodBody(methodSymbol, compilationState, diagsForCurrentMethod, out importChain, out originalBodyNested);
 
+                    if (body != null && methodSymbol.MethodKind == MethodKind.Constructor)
```

Comment:
>if (body != null && methodSymbol.MethodKind == MethodKind.Constructor) [](start = 20, length = 70)

It feels like this block should be moved next to the ```flowAnalyzedBody = FlowAnalysisPass.Rewrite(...``` statement.

Reply:
Added comment.